sdk/python/feast/templates/gcp/driver_repo.py

```diff
@@ -7,7 +7,7 @@
 # construction of feature vectors
 driver = Entity(
     # Name of the entity. Must be unique within a project
-    name="driver",
+    name="driver_id",
```

Comment:
was this intentional?

Reply:
Yeah, `get_online_features` uses entity name and the quickstart has `driver_id` in the input so the name has to be `driver_id` for the GCP template to work with the quickstart. When we change `get_online_features` to use the join_key instead of name this won't matter, but it's probably a good idea for the GCP template to match the local template here anyway